packages/beta/src/__tests__/api/dataPoints.int.spec.ts

```diff
@@ -0,0 +1,611 @@
+// Copyright 2020 Cognite AS
+
+import CogniteClient from '../../cogniteClient';
+import { DatapointAggregate, Timeseries, Datapoints } from '../../types';
+import { setupLoggedInClient } from '../testUtils';
+
+const DataPointStatus = {
+  good: {
+    code: 0,
+    symbol: 'Good',
+  },
+  uncertain: {
+    code: 1073741824,
+    symbol: 'Uncertain',
+  },
+  bad: {
+    code: 2147483648,
+    symbol: 'Bad',
+  },
+};
+
+describe('Datapoints integration test', () => {
+  const client = setupLoggedInClient();
+
+  let timeserie: Timeseries;
+  const timestampFrom = Date.now() - 60 * 60 * 48 * 1000;
+  const timestampTill = Date.now() - 60 * 60 * 24 * 1000;
+  const datapoints = [
+    {
```

Comment:
maybe we should not copy-paste all the existing tests? only add a new one with the statuses here?